package.json

```diff
@@ -47,12 +46,14 @@
     "graphql": "^16.8.1",
     "graphql-request": "^6.1.0",
     "graphql-tag": "^2.12.6",
+    "isbot": "^5.1.1",
     "jwt-decode": "^3.1.2",
     "lottie-react": "^2.4.0",
     "mdx-bundler": "^9.2.1",
     "mersenne-twister": "^1.1.0",
     "node-mailjet": "^6.0.4",
     "react": "18.2.0",
+    "react-code-blocks": "^0.1.6",
```

Comment:
where is this used? i must have missed it.

Reply:
Mantine dropped Prism (which is weird :/) so we had to pick a different code viewer library, and this was the smallest one I found... I had to customize it a bit though

https://github.com/pokt-foundation/portal-platform/blob/1554c2edb1167d3f17a23ef3c1b0d6ba0d8b0182/app/components/CodeSnippet/CodeSnippet.tsx